.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,5 +15,10 @@ Contributors guide: https://github.com/nodejs/node/blob/master/CONTRIBUTING.md
 - [ ] documentation is changed or added
 - [ ] commit message follows commit guidelines
 
+<!-- If this is your first contribution to Node.js, please also check the
```

Comment:
Can you move this to the bottom, so that repeat contributors don't have to deal with this every time. Also, is there any real point to making them checkboxes? We don't plan to enforce anything, right?